src/app/config/config.c

```diff
@@ -253,23 +254,45 @@ static config_abbrev_t option_abbrevs_[] = {
  * members with CONF_CHECK_VAR_TYPE. */
 DUMMY_TYPECHECK_INSTANCE(or_options_t);
 
-/** An entry for config_vars: "The option <b>name</b> has type
+/** An entry for config_vars: "The option <b>varname</b> has type
```

Comment:
Please document macro argument initvalue.

Reply:
This happens later in the #29211 branch, in c553750e32d1bf669a3e8308fa44319954a627ca when we move all these macros into src/lib/confmacros.h . I believe that is part of #30935.